demo/hello.ts

```diff
@@ -461,6 +451,22 @@ export class HelloApp {
         { name: 'formly-wrapper-horizontal', component: FormlyWrapperHorizontalLabel, types: ['horizontalInput'] },
         { name: 'panel', component: FormlyPanelWrapper },
       ],
+      defaultOptions: [
+        {
```

Comment:
defaultOptions is not meant to be set separately, its set as a part of the setType.